common/models/facility.js

```diff
@@ -454,17 +445,11 @@ module.exports = (Facility) => {
 
 
   Facility.facilitiesByTypeAndOwnership = async (districts, cb) => {
-    const IDs = await getDistrictsIDs(districts);
-
-    let facilities = null;
-    if (!IDs.length) {
-      facilities = await server.models.Facility.find();
-    } else {
-      facilities = await server.models.Facility.find({ where: { district_id: { inq: IDs } } });
-    }
-    const owners = await server.models.Owner.find();
-    const facilityTypes = await server.models.FacilityType.find();
-
+    const IDs = await getDistrictsIDs(districts)
+    const where = IDs.length ? { district_id: { inq: IDs } } : {}
+    const facilities = await Facility.find({ where })
```

Comment:
consider using a name like `facilitiesWithSelectedDistricts`... the principle here is that the variable name should represent what its returning